src/assets/js/archive.js

```diff
@@ -5,62 +5,109 @@ const releasesToShow = 99999;
+          this.closest('tr').remove();
           event.preventDefault();
         });
-        $('<tr>').append($('<td colspan="6"></td></tr>').append(showAll)).appendTo(table);
+
+        const row = document.createElement('tr');
+        const cell = document.createElement('td');
+        cell.colSpan = 6;
+        cell.appendChild(showAll);
+        row.appendChild(cell);
+        table.appendChild(row);
       }
 
-      const className = index >= releasesToShow ? 'overflow' : '';
+      const row = document.createElement('tr');
+      if(index >= releasesToShow) {
+        row.classList.add('overflow');
+      }
       const url = releases.base_url + '/' + release.archive;
-      const row = $('<tr />').addClass(className).appendTo(table);
-      const hashLabel = $('<span />').text(release.hash.substr(0, 7)).addClass('git-hash');
-      const downloadLink = $('<a />').attr('href', url).text(release.version);
-      const dartSdkVersion = $('<span />').text(
-        release.dart_sdk_version ? release.dart_sdk_version.split(' ')[0] : '-',
-      );
-      const dartSdkArch = $('<span />').text(
-        release.dart_sdk_arch ? release.dart_sdk_arch : 'x64',
-      );
+      table.appendChild(row);
+
+      const hashLabel = document.createElement('span');
+      hashLabel.textContent = release.hash.substr(0, 7);
+      hashLabel.classList.add('git-hash');
+
+      const downloadLink = document.createElement('a');
+      downloadLink.href = url;
+      downloadLink.textContent = release.version;
+
+      const dartSdkVersion = document.createElement('span');
+      dartSdkVersion.textContent = release.dart_sdk_version ? release.dart_sdk_version.split(' ')[0] : '-';
+
+      const dartSdkArch = document.createElement('span');
+      dartSdkArch.textContent = release.dart_sdk_arch ? release.dart_sdk_arch : 'x64';
+
       const date = new Date(Date.parse(release.release_date));
+
       const provenance = getProvenanceLink(os, release, date, channel);
-      $('<td />').append(downloadLink).appendTo(row);
-      $('<td />').append(dartSdkArch).appendTo(row);
-      $('<td />').append(hashLabel).appendTo(row);
-      $('<td />').addClass('date').text(date.toLocaleDateString()).appendTo(row);
-      $('<td />').append(dartSdkVersion).appendTo(row);
-      $('<td />').append(provenance).appendTo(row);
+    
+      
+      const cells = [
+        createTableCell(downloadLink),
+        createTableCell(dartSdkArch),
+        createTableCell(hashLabel),
+        createTableCell(date.toLocaleDateString(), 'date'),
+        createTableCell(dartSdkVersion),
+        createTableCell(provenance)
+      ];
+      
+      cells.forEach(function(cell) {
+        row.appendChild(cell);
+      });
     });
   }
 }
 
+function createTableCell(content, dataClass) {
+  console.log({content});
+  const cell = document.createElement('td');
+  if (dataClass) {
+    cell.classList.add(dataClass);
+  }
+  if(typeof content === 'string') {
```

Comment:
Consider adding an extra line to break this up a bit:

```suggestion

  if (typeof content === 'string') {
```